admin/commands/status_test.go

```diff
@@ -60,6 +60,7 @@ PMM Server:
 
 PMM Client:
 	Connected        : false
+	Connection uptime: 0
```

Comment:
Looking at this output it is not clear for what is `0` or `100`.
I can guess of course, but it should be clear. For example:
`Connection uptime: 0% since dd.mm.yyyy`

Additionally, it makes sense to add application start up

Reply:
@cezmunsta what do you think?

Reply:
@ritbl I would disagree. The whole point is to make it easy to detect issues with connectivity, making it a string does not do that.

Reply:
An example of its use:
```sh
function get_status {
    pmm-admin status --json | jq -r .pmm_agent_status.connection_uptime
}
​
function check_ready {
    test "$(get_status)" == "100" || return
    test "$(date +%s)" -gt "$((START_TS + 600))" || return
    printf "%s\tConnectivity restored, signalling stop\n" "$(date -Is)"
    touch /tmp/PMM-5005.abort
}
```